### PR TITLE
Rpi5 dev mmc follow up changes

### PR DESCRIPTION
**Pull request by GrygiriiS**
This PR is a set of follow up changes as per review of previous PR https://github.com/xen-troops/zephyr/pull/104

### CODE REVIEW

**Comment by GrygiriiS** [comment]
@xakep-amatop @oleksiimoisieiev

File: drivers/sdhc/sdhci_common.c [hunk 1]
```diff
@@ -352,6 +352,10 @@ static int sdhci_poll_data_complete(struct sdhci_common *sdhci_ctx, int32_t time
 			break;
```
Comment (oleksiimoisieiev) [inline drivers/sdhc/sdhci_common.c:352]: s/is error detected/if error detected/g

**Review by oleksiimoisieiev — COMMENTED** [review]
LGTM with a small comment:
`Acked-by: Oleksii Moisieiev <oleksii_moisieiev@epam.com>`

File: drivers/sdhc/sdhci_common.c (hunk 1 above)
Comment (oleksiimoisieiev) [reply drivers/sdhc/sdhci_common.c:352]: fixed

**Review by xakep-amatop — APPROVED** [review]
I'm okay with the changes present in this PR, but the https://github.com/xen-troops/zephyr/pull/104 still has open discussions.

I hope we can continue our communication to resolve the questions that have been raised there.

For these changes: `Acked-by: Mykola Kvach <mykola_kvach@epam.com>`

**Comment by GrygiriiS** [comment]
> I hope we can continue our communication to resolve the questions that have been raised there.

Sure

**Review by firscity — APPROVED** [review]
`Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>`

**Pull request merged**